recipe/meta.yaml

```diff
@@ -24,7 +24,6 @@ requirements:
     - python >=3.6
     - jigsaw 0.9.14
     - numpy
-    - pathlib
```

Comment:
There's no need for pathlib to be external -- it's now built in to python.  And it's causing me trouble because the migration of `mpas_tools` to `osx_arm64` is being held up waiting for the dummy `pathlib` conda package to get built.